.github/workflows/end2end-tests.yml

```diff
@@ -18,6 +18,6 @@ jobs:
           path: ${{ env.PACKAGEPATH }}
       - name: Build image
         working-directory: ${{ env.PACKAGEPATH }}
-        run: docker build . --file Dockerfile --tag xdsrelay
+        run: docker build . --file Dockerfile-e2e-tests --tag xdsrelay
```

Comment:
does running `make build-docker-image` work here?

Reply:
it does (as`--file` argument allows to provide custom file name)

Reply:
In that case how about we just run `make build-e2e-tests-docker-image` here?

Reply:
cannot yet explain the reason, but suggested change makes the next step "run e2e tests inside container" fail with error:
`Run docker run -v /home/runner/work/xds-relay/xds-relay/go/src/github.com/envoyproxy/xds-relay:/xds-relay xdsrelay:latest make e2e-tests
Unable to find image 'xdsrelay:latest' locally
docker: Error response from daemon: pull access denied for xdsrelay, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.`

Reply:
interesting. Github Actions is probably detects the use of `docker build` in the command and sets up the right permissions. That said, I couldn't find anything in the docs though. Feel free to revert https://github.com/envoyproxy/xds-relay/pull/133/commits/48087da1a2280021dbfa0d4fbd6f846fd50d3e73.